src/Integration/Telemetry/TelemetryPayload.cs

```diff
@@ -49,6 +49,9 @@ public sealed class TelemetryPayload
         [JsonProperty("connected_mode_used")]
         public bool IsUsingConnectedMode { get; set; }
 
+        [JsonProperty("legacy_connected_mode_used")]
+        public bool IsUsingLegacyConnectedMode { get; set; }
```

Comment:
I agree with the PM that this is what he wants for now. We won't be able to tell if someone is using both legacy connected mode and new connected mode, but that's acceptable for now.